src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseEditableBlock.tsx

```diff
@@ -53,18 +54,18 @@ function EditableBlockRenderer({
     case 'MarkDownBlock':
       return (
         <EditableContentBlock
-          editable={editable}
+          editable={editable && !isBrowser('IE')}
```

Comment:
Why the checks against IE?  Is this a limitation that's been agreed with DfE?  I'm guessing that Analysts have been instructed to use Edge or Chrome?

Reply:
It's OK - I've got the answer from Cam thanks!  All good